Yoast/wordpress-seo
# Indexable search results

## Summary

Adds builder and watcher for creating and updating the indexable for search results.

This PR can be summarized in the following changelog entry:

* N/A

## Test instructions
This PR can be tested by following these steps:

* Make sure you have something that presents the title.
* Go to the search results, you should see that title presented.
* Go to search appearance and change the title of your search results.
* Go to your search results again, you should see the title has changed to the new value.

## UI changes

## Documentation

## Quality assurance
